src/lib/spinel/radio_spinel.cpp

```diff
@@ -111,7 +111,8 @@ RadioSpinel::RadioSpinel(void)
     , mVendorRestorePropertiesCallback(nullptr)
     , mVendorRestorePropertiesContext(nullptr)
 #endif
-    , mEnableRcpTimeSync(false)
+    , mTimeSyncFeatureEnabled(false)
```

Comment:
What about naming as:
* `mTimeSyncEnabled` -> `mTimeSyncOn`
* `mTimeSyncFeatureEnabled` -> `mTimeSyncEnabled`

Current naming also works for me.